standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/RawStore.java

```diff
@@ -140,17 +140,6 @@ public interface RawStore extends Configurable {
    */
   boolean openTransaction();
 
-  /**
-   * Opens a new one or the one already created. Every call of this function must
-   * have corresponding commit or rollback function call.
-   *
-   * @param isolationLevel The transaction isolation level. Only possible to set on the first call.
-   * @return an active transaction
-   */
-  default boolean openTransaction(String isolationLevel) {
-    throw new UnsupportedOperationException("Setting isolation level for this Store is not supported");
-  }
-
```

Comment:
`RawStore` is a user-facing (`metastore.rawstore.impl`) public interface so removing this default method is a breaking change. I guess there are not many users relying on this so it should be rather safe to drop it. However, I feel that dropping support for transactions with `isolationLevel` should have its own dedicated JIRA case unless its really tight to this fix.

Reply:
+1

Reply:
hi @pvary, do you recall why a new signature for `openTransaction` with `isolation` level param was introduced?

Reply:
oh, we removed the use of it in https://github.com/apache/hive/pull/5129/files#diff-394eb2401c3389f5de729f3f1c6a423aaf0041d89d406212b14abf90bc650358L189, so should be fine to drop from the interface